community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+
+If we see posts that we judge to not meet these guidelines, we will actively remove them, but will also let you know privately why we're doing so. 
+
+Everything in this forum is also covered by our [Code of Conduct](https://exercism.org).
+
+<a name="encourage"></a>
+
+## [Encourage each other](#improve)
+
+This is first and foremost an educational space - a space to learn and share ideas, to uplift each other. It is **not** a space to prove yourself, show off your knowledge, compete with others, create a sense of “us vs them”, or put others down. We take a strict approach to this, and so topics that would be considered ok in other forums are not considered ok here. 
+
+Some examples:
+- A discussion exploring what the tradeoffs would be between using C# vs F# could be a positive and educational experience. However, a comment saying “C# is rubbish because …” is not helpful, does not add to the educational experience and would be deleted.
+- Any conversations around topics such as politics or religion are not appropriate in this space. Neither are topics that are fought with a religious fervor (e.g. whether VIM or Emacs is better).
+- Any sort of comment that implies “noobs do …” or “real developers use …” will be deleted.
+
+<a name="improve"></a>
+
+## [Improve the Discussion](#improve)
+
+Help us make this a great place for discussion by **always adding something positive to the discussion**. If you are not sure your post adds to the conversation, think over what you want to say and try again later.
+
+One way to improve the discussion is by discovering ones that are already happening. Spend time browsing the topics here before replying or starting your own, and you’ll have a better chance of meeting others who share your interests.
+
+The topics discussed here matter to us, and we want you to act as if they matter to you, too. Be respectful of the topics and the people discussing them, even if you disagree with some of what is being said.
+
+<a name="agreeable"></a>
+
+## [Be Agreeable, Even When You Disagree](#agreeable)
+
+You may wish to respond by disagreeing. That’s fine. But remember to _criticize ideas, not people_. Please avoid:
+
+* Name-calling
+* Ad hominem attacks
+* Responding to a post’s tone instead of its actual content
+* Knee-jerk contradiction
+
+Instead, provide thoughtful insights that improve the conversation.
+
+Write your opinions **as opinions**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
```

Comment:
```suggestion
Write your opinions **as I-messages**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
```

I feel that explicitly refering to I-messages is clearer here because it's not uncommon to assume that everything one writes in a forum is an opinion. Alternatively maybe something like this could underline that such an assumption should not be made:

```suggestion
Be **explicit** about your opinions **being opinions**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
```

Reply:
Thanks. I prefer this second suggestion to the first as I don't know what "I messages are" (I can guess, but others might not guess). @kytrinyx Any thoughts here?

Reply:
I've also never heard of an "I-message", so that might be something that people in general don't know about.

Reply:
I hadn't heard of it, either. I don't know if this is something that a lot of people haven't heard of, or it just happens to be me and Erik. I like the wikipedia article on it, though.

If it's likely that a lot of people haven't heard about it, I would lean towards not introducing the idea here, because it means we're potentially leading people away from this article, whereas I really, really want people to read the whole thing once they've started, because it seems less likely that they'll finish if they click away (and we want people to have read it before they start posting in the forum).

Reply:
Whatever we do, I do really like the _extra explicitness_ here that Sascha is trying to add. The second suggestion would be much better than the current doc imo.

Reply:
I also never heard of I-messages before.